CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 # Change Log (Release Notes)
 
 All _notable_ changes to this project will be documented in this file (`CHANGELOG.md`).
-This project adheres to [the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme) (or at least we try).
+
+This project adheres to [Semantic Versioning 2](http://semver.org/spec/v2.0.0.html), the BigchainDB **public API**:
+* Includes:
+  * This definition of **public API**.
+  * The Command Line Interface.
+  * The transaction schema.
+  * All the endpoints exposed by the HTTP API (including payloads sent to and received from them).
```

Comment:
It's recursive, the idea is that the definition of **public API** in `CHANGELOG.md` is part of the **public API** itself. If we don't have this rule, then someone can change **what we mean** as **public API** and mess everything up.